cypress/integration/gift_card_spec.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="Cypress" />
+import { HomePo } from '../support/home.po';
+import { GiftCardsPo } from '../support/giftCard.po';
+
+
+describe('100$ gift card add to cart', () => {
+    const homePage = new HomePo();
+    const giftPage = new GiftCardsPo();
+
+    beforeEach(() => cy.visit(homePage.pageUrl));
+
+    it('gift card add to cart', () => {
+
+        const successMessage = 'Added to Cart';
+        cy.viewport(1175, 600);
+
+        giftPage.clickOnElem(giftPage.giftCardLink);
+        giftPage.isURLcontain(giftPage.giftCardUrl);
+        giftPage.clickOnElem(giftPage.eGift);
+        giftPage.isElemVisible(giftPage.prise100);
```

Comment:
prise100 => price100
unnecessary check, because if this elem will be not visible, you could not click on it

Reply:
Then user redirected to page with product route - this check not covered, should be smth like this:
cy.url().should('include', '/product/');